### DISCUSSION TITLE
What is the maximum size of an office file to be able to convert it to PDFA using the Gotenberg image in docker? and How can I change or configure its value?

### DISCUSSION THREAD

**Discussion by gloria256** [gotenberg/gotenberg#696] [Q&A] [answered]
It happens that when you try to send an Office file that is too large (for example, 20 MB) to convert to PDFA, the Gotenberg7 service installed in Docker returns that it cannot process the request.

I appreciate if anyone can support me. Thank you so much

**Comment by gulien** [comment, marked as answer]
Hello @gloria256,

LibreOffice does struggle with "large" documents. Gotenberg just waits for LibreOffice to finish its conversion, but if it takes too much time, it returns a `503` HTTP code. You may try to increase the `--api-timeout` (for instance `300s`) but it may not be enough.

**Reply by gloria256** [reply]
thanks for everything it worked for us increasing the wait time to 600 seconds
